Collection_Part_2/src/de/telran/collectionP2.java

```diff
@@ -0,0 +1,31 @@
+package de.telran;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+
+public class collectionP2 {
+
+    public static List<String> removeDuplicates(List<String> input){
+
+        HashSet<String> hs = new HashSet<>();
+
+        hs.addAll(input);
+        input.clear();
```

Comment:
It is better not to modify input list but rather create a new one, which you return. You add all to this new list from the set.